### DISCUSSION TITLE
A adult/porn subscription list

### DISCUSSION THREAD

**Discussion by Claromale** [iorate/ublacklist#652] [General]
Hey i just wondering if someone here made a porn/adult subscription list for uBlockList. I'de like to have one for me please !

**Comment by aquelepedroaugusto** [comment]
Hey there! I don't think this is in the scope of this repository.

But anyways, I just did a quick check on multiple search engines including: Google, DuckDuckGo, Bing, Brave, Ecosia and some SearXNG instances.

All of them seem to have a "Safe search" functionality that filters "adult content", though the SearX one doesn't seem to be as good.

Do you have a reason why those tools aren't enough for you?

**Reply by realAzazello** [reply]
@Claromale : I've [a personal uB-list](https://gist.github.com/realAzazello/22adb48881f4fcdf9bb20b983593340f) that has a "p0rn" section. Feel free to use any of the filters in there to your preferences.

**Comment by Claromale** [comment]
hey Aug ! 
Safe Search is a great tool, but you can disable it. With Stayfocusd, i can disable safe research options, but on the page you can disable safe research directly (the button appear on the top right of the page)

Like maybe it can be deleted by UBlock ?

**Comment by aquelepedroaugusto** [comment]
I did some research, and _maybe_ [this extension](https://github.com/serv-inc/safe-search) might help.

It enforces safe search on multiple search engines, including ones that aren't supported by uBlacklist.

Now, about the button on the page, you can use [uBlock Origin](https://ublockorigin.com/) to remove it.

---

This is an interesting topic.

When it comes to your suggestion (having an adult content block list), I imagine that such list would be gigantic, and it would probably slow down your browser if you had to go through it at every search query you make.

The aforementioned safe search extension does something that seems to be in line with uBlacklist, and maybe it could be useful to have as part of the project. (If the maintenance burden isn't too big).

Now granted, I'm still very new to this codebase and I don't know what  the main developer thinks about it, so my opinion doesn't have much weight here.

**Comment by iorate** [comment]
While I do optimize for performance even with large lists, I agree that creating such subscription lists is out of scope for this project.

However, I think it would be fine for users to discuss creating such lists among themselves. I'll move this discussion to Discussions. That would be a more appropriate place for community members to discuss such lists.

**Reply by herrbrixter** [reply]
I agree. Porn blocking is out of scope.

**Comment by ghost** [comment]
Hope this helps. Subscribe to either of these: 

1. https://nsfw.oisd.nl/ublacklist
2. https://nsfw-small.oisd.nl/ublacklist

Submit missed domains here: https://oisd.nl/nsfw_reporter
Let oisd handle the rest
